ui/packages/catalog-ui-search/src/main/webapp/react-component/container/map-actions/map-actions.tsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import { hot } from 'react-hot-loader'
+import * as React from 'react'
+const wreqr = require('../../../js/wreqr.js')
+const _ = require('underscore')
+
+type Props = {
+  model: Backbone.Model
```

Comment:
❓ would it be better to pass in the properties that you need from the model instead of the model itself? It doesn't look like you are listening to anything on it.

Reply:
https://github.com/codice/ddf/pull/4439/files#diff-9737763c0e03df232ce865ef2ea6390bR96 the `currentOverlayUrl` needs to be set on the model